components/script/dom/htmlstyleelement.rs

```diff
@@ -280,4 +280,17 @@ impl HTMLStyleElementMethods for HTMLStyleElement {
     fn GetSheet(&self) -> Option<DomRoot<DOMStyleSheet>> {
         self.get_cssom_stylesheet().map(DomRoot::upcast)
     }
+
+    // https://html.spec.whatwg.org/multipage/#dom-style-disabled
```

Comment:
```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-style-disabled>
```

Reply:
OK, I just copied this from the GetSheet above, so I will change that too.

Reply:
Yeah, this is quite common in the script crate. A lot of the old documentation isn't proper rustdoc.